peas/weather.py

```diff
@@ -604,20 +587,19 @@ def get_wind_speed(self, n=3):
         return self.wind_speed
 
     def send_message(self, msg, channel='weather'):
-        if self.messaging is None:
-            self.messaging = PanMessaging.create_publisher(6510)
-
-        self.messaging.send_message(channel, msg)
+        super.send_message()
```

Comment:
If you don't actually do anything in the method you override you can just leave the whole method out and it will use the base class by default.